package.xml

```diff
@@ -38,7 +38,7 @@
   <license>New BSD</license>
   <url type="website">http://ceres-solver.org/</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
> Just so I understand, why would it work to leave buildtool_depend as catkin? Wouldn't ament look at that, try to satisfy the dependency, and then fail?

Nope, ament doesn't care about dependencies it cannot find. `ament_tools` will use dependencies it can find in the same workspace for topological order of the builds, but `ament_cmake` doesn't need it at all. `ament_cmake_auto` might try to use it to `find_package` something, but it will not fail if it is not found.

So technically you can put anything in a depend tag and it won't matter locally. However, when making a binary package bloom will try to figure out what dependency to put in the `debian/controls` file and fail if there is no `rosdep` key or released ros package to match it with.